.github/workflows/forest.yml

```diff
@@ -44,6 +48,9 @@ jobs:
     name: Build Ubuntu (Forest daemon)
     runs-on: ubuntu-latest
     steps:
+      - name: Show IP
```

Comment:
Consider adding a comment here saying this aims to find the nearest S3 space for storing cache files.

Reply:
fixed